openmc/plots.py

```diff
@@ -67,6 +67,16 @@ class Plot(object):
     col_spec : dict
         Dictionary indicating that certain cells/materials (keys) should be
         colored with a specific RGB (values)
+    level : int
+        Universe depth to plot at
+    meshlines_type : {'tally', 'entropy', 'ufs', 'cmfd'}
+        The type of the mesh to be plotted
+    meshlines_id : int
+        ID for the mesh specified on ``tallies.xml`` that should be plotted
+    meshlines_linewidth : int
+        Pixels of linewidth to specify for the mesh boundaries
+    meshlines_color : Iterable of int
+        Color for the meshlines boundaries by RGB.
```

Comment:
So as not to have the number of attributes explode, perhaps we should just have these be in a dictionary, e.g.
```Python
plot.meshlines = {
  'type': 'tally',
  'id': 3,
  'linewidth': 2,
  'color': (122, 122, 0)
}
```